src/components/label.js

```diff
@@ -0,0 +1,222 @@
+import React, { forwardRef, useState } from 'react'
+import PropTypes from 'prop-types'
+
+import { concatStateColor, concatTargetColor } from '../utils/concat-color'
+
+import { ButtonBase } from '../base-components'
+import { makeStyles } from '../utils/make-styles'
+import { getTailwindConfigColor } from '../utils/tailwind-config-color'
+
+
+const _labelSize = (size) => {
+  let labelSize = ''
+
+  switch(size) {
+  case 'lg':
```

Comment:
For readability, all the block code for switch should be further indented. Like here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

Reply:
it is from lint --fix tho do you know how I can make not complaining about it?